content/en/examples/concepts/policy/limit-range/example-conflict-with-limitrange-cpu.yaml

```diff
@@ -2,6 +2,7 @@ apiVersion: v1
 kind: Pod
 metadata:
   name: example-conflict-with-limitrange-cpu
+  namespace: default
```

Comment:
Can use other than default, like dev / prod

Reply:
The problem with using other namespace than default is when user copies the manifest and executes it, they will potentially run into namespace not found error. 
Although it might be a simple error which users can fix themselves, this will disrupt the user's workflow as the namespace creation is not mentioned anywhere before.

Reply:
@utkarsh-singh1  wdyt? Do you still recommend specifying dev/prod namespace? 
May be we can add a note to create the namespace before applying the yaml?

Reply:
IMO

To keep the concept simple, I think it's okay to keep namespace as the default namespace and include a comment in the YAML manifest indicating that the namespace is implied.

@subhals, @utkarsh-singh1